Dockerfile

```diff
@@ -123,7 +138,7 @@ COPY health_check.sh /
 EXPOSE 9000
 USER ${GRAYLOG_USER}
 VOLUME ${GRAYLOG_HOME}/data
-ENTRYPOINT ["/docker-entrypoint.sh"]
+ENTRYPOINT ["/tini", "--", "/docker-entrypoint.sh"]
```

Comment:
This may end up being `/opt/tini`, depending on the result of the discussion above